docs2/site/docs/migrations/migration5.md

```diff
@@ -551,6 +551,36 @@ be properly serialized. Now you may serialize a `ExecutionError` instance direct
 be handled by the specified `IErrorInfoProvider` and serialized correctly. This can be useful
 when implementing the newer `graphql-transport-ws` WebSockets protocol.
 
+### 18. Subscription support improved
```

Comment:
Take a look at
> ### 29. `SubscriptionDocumentExecuter` and `.AddSubscriptionDocumentExecuter()` have been deprecated.

Reply:
I will review/update.

Reply:
Removed 29.  Made another small change to align with this PR.